pydis_site/templates/home/timeline.html

```diff
@@ -0,0 +1,449 @@
+{% extends 'base/base.html' %}
+{% load static %}
+
+{% block title %}Timeline{% endblock %}
+{% block head %}
+<link rel="stylesheet" href="{% static "css/home/timeline.css" %}">
+<link rel="stylesheet" href="{% static "css/home/index.css" %}">
+{% endblock %}
+
+{% block content %}
+{% include "base/navbar.html" %}
+
+<section class="cd-timeline js-cd-timeline">
+    <div class="container max-width-lg cd-timeline__container">
+        <div class="cd-timeline__block">
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Python Discord is created</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
+
+                <div class="flex justify-between items-center">
+                    <span class="cd-timeline__date">Jan 8th, 2017</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Python Discord hits 1,000 members</h2>
+                <p class="color-contrast-medium">lorem ipsum</p>
+
+                <div class="flex justify-between items-center">
+                    <span class="cd-timeline__date">Nov 10th, 2017</span>
+                </div>
+            </div> <!-- cd-timeline__content -->
+        </div> <!-- cd-timeline__block -->
+
+        <div class="cd-timeline__block">
+            <!-- ... -->
+            <div class="cd-timeline__img cd-timeline__img--picture">
+                <img src={% static "images/timeline/cd-icon-picture.svg" %} alt="Picture">
+            </div> <!-- cd-timeline__img -->
+
+            <div class="cd-timeline__content text-component">
+                <h2>Our logo is born. Thanks Aperture!</h2>
```

Comment:
Maybe @Aperture for consistency?
```suggestion
                <h2>Our logo is born. Thanks @Aperture!</h2>
```